.github/workflows/build-and-publish-release.yml

```diff
@@ -0,0 +1,102 @@
+name: Build and Publish Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: "Version. Please DO NOT include the 'v' prefix"
+        required: true
```

Comment:
I think @mattray had an idea that we could have a workflow like this trigger when a version tag appears on the repo -- this condition: https://github.com/kubecost/kubectl-cost/blob/a9d05d340121a823232b6fe05be6a3d35ea72c78/.github/workflows/build-release.yaml#L4-L6
```
on:
  push:
    tags:
      - 'v*'
```

Then all you have to do is add a `v.....` tag to the repo and the workflow will automatically kick off. If that's the long-term direction we want to go, it might be valuable to start there as well.

Reply:
ok, I can add a new pr with that, since I already put the tag out I need to get this one built. I'll throw up a second pr later.